app/src/main/AndroidManifest.xml

```diff
@@ -13,6 +13,9 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.CoachInfoMalawi"
         tools:targetApi="31">
+        <activity android:name=".Axa.AxaActivity"
```

Comment:
Why did you change the name to Axa ?

Reply:
Oh this was in the Axa folder. Because I made too many activities, I thought I would separate them according to the info they contain

Reply:
Got it, I think with a JSON file, you may not need this anymore. Lets see.

Reply:
Sure thing, send it and guide me through. I was watching a video on this but I still found it confusing

Reply:
Cool, I am watching a youtube video on it. Kinda confusing but still trying to understand it